packages/wallet-sdk/src/connection/DiagnosticLogger.ts

```diff
@@ -17,7 +16,7 @@ export type LogProperties = {
   onlineGuests?: number; // number of online guests (should be 0 or 1)
   sessionIdHash?: string; // anonymous session id for debugging specific sessions
   sessionMetadataChange?: string; // change in session metadata
-  state?: ConnectionState;
+  state?: boolean;
```

Comment:
btw, in case you're wondering,
the change maintains functional integrity for logging purposes, because
`ConnectionState` was an enum with DISCONNECTED as its first value => having a raw value of 0